scripts/src/bus.lua

```diff
@@ -5520,6 +5520,20 @@ if (BUSES["TMC600"]~=null) then
 	}
 end
 
+---------------------------------------------------
+--
+--@src/devices/bus/technics/kn5000_extension.h,BUSES["TECHNICS"] = true
+---------------------------------------------------
+
+if (BUSES["TECHNICS"]~=null) then
```

Comment:
I'd call this bus something like "kn5000" unless all Technics keyboards used it.

Reply:
I do see the actual files are named how I'd expect.  Maybe a structure like bus/technics/kn5000/kn5000_extension.cpp so it's clear in the future what's going on.

Reply:
@rb6502, yes, I can confirm everything is fine. Your suggested changes are implemented and in good shape.